contents/blog/best-open-source-analytics-tools.md

```diff
@@ -198,6 +213,6 @@ Like Plausible Analytics, Pirsch is focused on users who only need to track basi
 ## Open source license
 Pirsch is completely open source, under a GNU AGPLv3 license. Pricing for the full version is based solely on pageviews, meaning it can be deployed across multiple sites at no extra cost. Check [the Pirsch repo](https://github.com/pirsch-analytics/pirsch) for more info. 
```

Comment:
Technically, AGPLv3 is _not_ open source, since it's a copyleft license. This means you can't for example build a paid product on top of it.

Reply:
(internet is conflicting if you look this up, but generally this introduces lots of issues for ie enterprises trying to use it in production)

Reply:
Should we remove Pirsch from this list, then? Feels like it might be a bit borderline to have it in there. There are plenty of alternatives we could use instead.

Reply:
probably worth a footnote or perhaps  a disclaimer like "also worth a look"?

Reply:
Hmnm, I'm siding on removing it for now. Feels like there are too many caveats to explain.